### DISCUSSION TITLE
Unix socket permissions setting?

### DISCUSSION THREAD

**Discussion by rob4226** [verdaccio/verdaccio#2373] [Q&A]
When using a unix socket, is there a setting for the permissions used for the socket?  I can't seem to find it.  When using with a reverse proxy it is usually necessary to give the web server user, group ownership of the socket, and give it write permissions. I can handle the group owner through the parent directory's group owner, but cannot control the write permission, and nginx needs write permission or will get a permission denied error.

```yaml
listen:
 - unix:/tmp/verdaccio.sock    # unix socket
```

Is there a permissions setting? Thanks

**Comment by juanpicado** [comment]
@Rob4226 I haven't played around much with something different than just a http port (I guess you use it because is faster), I've got some reports with `unix` but I'm afraid cannot help here.
